### PR TITLE
filmizletv.vip

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/141894

### Add your comment and screenshots

None.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by zloyden — COMMENTED** [review]
I also see video ad.

File: TurkishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -39,6 +39,8 @@ trgoals185.xyz,trgoals186.xyz,trgoals187.xyz,trgoals188.xyz,trgoals189.xyz,trgoa
...
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/141894
+filmizletv.vip#%#//scriptlet('prevent-setTimeout', 'filmizletv.vip/uploads/Psk/')
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply TurkishFilter/sections/general_extensions.txt:43]: To be satisfied the following sentence as the reporter said:
> Also invisible ads on the sides.

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -68,6 +68,9 @@ bursa.com,sozcu.com.tr,megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.
 /service/advertchannels?$xmlhttprequest,redirect=nooptext,domain=mardinhaber.com.tr|gidahatti.com|bolugundem.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
+||filmizletv.vip/Superbet$image
+||filmizletv.vip/suberbet$image
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply TurkishFilter/sections/specific.txt:72]: Pleas just a sec.

File: TurkishFilter/sections/specific.txt [hunk 3]
```diff
@@ -68,8 +68,7 @@ bursa.com,sozcu.com.tr,megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.
 /service/advertchannels?$xmlhttprequest,redirect=nooptext,domain=mardinhaber.com.tr|gidahatti.com|bolugundem.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
-||filmizletv.vip/Superbet$image
-||filmizletv.vip/suberbet$image
+/^https:\/\/filmizletv\.vip\/su(b|p)erbet/
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:71]: add domain resctriction

Comment (piquark6046) [reply TurkishFilter/sections/specific.txt:72]: `/^https:\/\/filmizletv\.vip\/`, a part of the regexp, does already restrict the domains that the regexp applies.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**